canonical/snapcraft.io
# chore: fix linting errors in /static/js/libs and formatting errors

## Done
- opened a branch from #4777 
- fix the existing formatting errors
- fix the linting errors in `snapcraft.io/static/js/libs`

## How to QA
- `lint-js` should be failing as there are other linting errors left to be fixed
- starting from [this line](https://github.com/canonical/snapcraft.io/pull/4790/files#diff-015c77ea9c404c78ce97835474293c38b268481a66424581b4636d53e4b05631R4), check those files below:
`static/js/libs/__tests__/iframeSize.test.ts`
`static/js/libs/debounce.ts`
`static/js/libs/__tests__/throttle.test.ts`
`static/js/libs/__tests__/debounce.test.ts`

## Testing
- [x] No testing required (explain why): fixing linting errors

## Issue / Card
Fixes #https://warthogs.atlassian.net/browse/WD-13579
